include/dos_inc.h

```diff
@@ -287,40 +331,37 @@ class DOS_PSP :public MemStruct {
 		SetPt(segment);
 		seg = segment;
 	}
-	void	MakeNew				(Bit16u memSize);
-	void	CopyFileTable		(DOS_PSP* srcpsp,bool createchildpsp);
-	Bit16u	FindFreeFileEntry	(void);
-	void	CloseFiles			(void);
-
-	void	SaveVectors			(void);
-	void	RestoreVectors		(void);
-
-	void SetSize(uint16_t size) { sSave(sPSP, next_seg, size); }
-	uint16_t GetSize() { return (uint16_t)sGet(sPSP, next_seg); }
 
-	void SetEnvironment(uint16_t eseg) { sSave(sPSP, environment, eseg); }
-	uint16_t GetEnvironment() { return (uint16_t)sGet(sPSP, environment); }
+	void MakeNew(uint16_t mem_size);
+	void CopyFileTable(DOS_PSP *srcpsp, bool createchildpsp);
+	void CloseFiles();
 
-	uint16_t GetSegment() { return seg; }
+	void SaveVectors();
+	void RestoreVectors();
 
-	void	SetFileHandle		(Bit16u index, Bit8u handle);
-	Bit8u	GetFileHandle		(Bit16u index);
+	uint16_t GetSegment() const { return seg; }
 
-	void SetParent(uint16_t parent) { sSave(sPSP, psp_parent, parent); }
-	uint16_t GetParent() { return (uint16_t)sGet(sPSP, psp_parent); }
+	void SetFileHandle(uint16_t index, uint8_t handle);
+	uint8_t GetFileHandle(uint16_t index) const;
+	uint16_t FindFreeFileEntry() const;
+	uint16_t FindEntryByHandle(uint8_t handle) const;
 
-	void SetStack(RealPt stackpt) { sSave(sPSP, stack, stackpt); }
-	RealPt GetStack() { return sGet(sPSP, stack); }
+	void SetSize(uint16_t size) { SSET_WORD(sPSP, next_seg, size); }
+	void SetInt22(RealPt int22pt) { SSET_DWORD(sPSP, int_22, int22pt); }
+	void SetParent(uint16_t parent) { SSET_WORD(sPSP, psp_parent, parent); }
+	void SetEnvironment(uint16_t env) { SSET_WORD(sPSP, environment, env); }
+	void SetStack(RealPt stackpt) { SSET_DWORD(sPSP, stack, stackpt); }
+	bool SetNumFiles(uint16_t file_num);
+	void SetFCB1(RealPt src);
+	void SetFCB2(RealPt src);
+	void SetCommandTail(RealPt src);
 
-	void SetInt22(RealPt int22pt) { sSave(sPSP, int_22, int22pt); }
-	RealPt GetInt22() { return sGet(sPSP, int_22); }
+	uint16_t GetSize() const { return SGET_WORD(sPSP, next_seg); }
+	RealPt GetInt22() const { return SGET_DWORD(sPSP, int_22); }
+	uint16_t GetParent() const { return SGET_WORD(sPSP, psp_parent); }
+	uint16_t GetEnvironment() const { return SGET_WORD(sPSP, environment); }
+	RealPt GetStack() const { return SGET_DWORD(sPSP, stack); }
```

Comment:
Suggest pairing up the rest of these in Setter-and-Getter groups, like you've done further up for Save & Restore Vectors, and Set and Get FileHandle.